pkg/cmd/roachtest/option/options.go

```diff
@@ -30,6 +30,15 @@ func DefaultStartOpts() StartOpts {
 	return startOpts
 }
 
+// DefaultStartOptsInMemory returns a StartOpts populated with default values,
+// and with in-memory storage
+func DefaultStartOptsInMemory() StartOpts {
+	startOpts := StartOpts{RoachprodOpts: roachprod.DefaultStartOpts()}
+	startOpts.RoachprodOpts.ScheduleBackups = true
+	startOpts.RoachprodOpts.ExtraArgs = append(startOpts.RoachprodOpts.ExtraArgs, "--store=type=mem,size=0.3")
```

Comment:
Maybe add a comment to explain "0.3 means 30% of available RAM".

Reply:
done